backend/src/main/java/com/bakdata/conquery/apiv1/FilterSearch.java

```diff
@@ -126,38 +125,28 @@ public void updateSearch() {
 					for (Searchable searchable : collectedSearchables) {
 
 						service.submit(() -> {
-							final Stream<FEValue> values = searchable.getSearchValues(getParserConfig(), storage);
 
```

Comment:
Mir ist gestern aufgefallen, dass sich der Job nicht canceln lässt, kannst du das heir in einen separaten Job heben und dann eine cancel methode implementieren? Es reicht ein shutdownNow zu machen in der join-loop und dann zu exiten wenn der rest durchgelaufen ist (vlt auch ein clear?) das hätte mir gestern einiges an zeit gespart :D